operator/cmd/mesh/operator-init.go

```diff
@@ -56,16 +47,11 @@ func addOperatorInitFlags(cmd *cobra.Command, args *operatorInitArgs) {
 	cmd.PersistentFlags().StringVarP(&args.inFilename, "filename", "f", "", "Path to file containing IstioOperator custom resource")
 	cmd.PersistentFlags().StringVarP(&args.kubeConfigPath, "kubeconfig", "c", "", "Path to kube config")
 	cmd.PersistentFlags().StringVar(&args.context, "context", "", "The name of the kubeconfig context to use")
-	cmd.PersistentFlags().DurationVar(&args.readinessTimeout, "readiness-timeout", 300*time.Second, "Maximum seconds to wait for the Istio operator to be ready."+
```

Comment:
why we remove the wait flag?

Reply:
i think it's confusing for this command. we don't wait for istio itself (and shouldn't because the controller works as a reconciler) and the controller container is so simple there's little point in exposing a wait flag especially for that.